tests/end-to-end/upgrade/README.md

```diff
@@ -46,6 +46,7 @@ Use the following flags to configure the application:
 |-----|:---------:|------------|
 | **action** | YES | Defines what kind of action to execute. The possible values are `prepareData` and `executeTests`. |
 | **verbose** | NO | Prints logs for all tests. |
+| **testName** | NO | Limit testexecution to the given test, if not specified all tests will be executed |
```

Comment:
```suggestion
| **testName** | NO | Limit test execution to the given test. If not specified, all tests will be executed. |
```

However, if test execution is a technical term and should be one word, it can be ``testexectution`` in code format.